### PR TITLE
Add Script Command: imessage-2fa

### PR DESCRIPTION
**Pull request by crstauf**
Get most recent two-factor authentication code from iMessages.

### CODE REVIEW

File: communication/imessage-2fa.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# @raycast.schemaVersion 1
+# @raycast.title Get latest 2FA code from iMessages
+# @raycast.mode compact
+
+# Optional parameters:
+# @raycast.author Caleb Stauffer
+# @raycast.authorURL https://github.com/crstauf
+# @raycast.description Get most recent two-factor authentication code from iMessages.
+# @raycast.packageName Communication
+# @raycast.icon 🔐
```
Comment (unnamedd) [inline communication/imessage-2fa.sh:13]: 🤘
so much better than that one I suggested! haha 😁

File: communication/imessage-2fa.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Get latest 2FA code from iMessages
+# @raycast.mode compact
+
+# Optional parameters:
+# @raycast.author Caleb Stauffer
+# @raycast.authorURL https://github.com/crstauf
+# @raycast.description Get most recent two-factor authentication code from iMessages.
+# @raycast.packageName Communication
```
Comment (unnamedd) [inline communication/imessage-2fa.sh:12]: If the folder (communication) is the same of the value set in metadata `packageName` you don’t need to configure it, Raycast and the Toolkit will not use it.

**Review by unnamedd — APPROVED** [review]

**Comment by crstauf** [comment]
@unnamedd and I (mostly @unnamedd) are working on making this just a single Bash script, rather than using PHP.

**Comment by shpakovski** [comment]
@crstauf We prefer “Rebase and merge”. Could you please check the conflict?

**Comment by crstauf** [comment]
@shpakovski GitHub is not indicating any conflict... what is it you're seeing?

**Comment by shpakovski** [comment]
Oh… I see this:

image

**Comment by shpakovski** [comment]
@crstauf Could you please fetch latest `master` and rebase on top, then force-push?

**Comment by shpakovski** [comment]
Wonderful, cheers!

**Pull request merged**